### ISSUE TITLE
Longer hostnames cause wrapping issues in Server status

### ISSUE THREAD

**Issue by ShadyShell** [Wruczek/ts-website#95]
I would prefer to give my users a URL rather than an IP address to connect to, however my domain name is rather long and I run into the issues pictured in the screenshots below.

Normal:
![screenshot_3](https://user-images.githubusercontent.com/5232510/45866755-66cf7600-bd47-11e8-90bb-29b7fba18407.png)

Longer hostnames:
![screenshot_2](https://user-images.githubusercontent.com/5232510/45866754-66cf7600-bd47-11e8-925c-ee45bda216fe.png) ![screenshot_1](https://user-images.githubusercontent.com/5232510/45866696-3982c800-bd47-11e8-87fa-4db3918567ec.png)

**Comment by Wruczek** [comment]
Thanks for reporting!
It's a very specific bug that probably affects small number of ts-website installations.

1.0 is kinda in the end-of-life state right now, so i think I will leave it as wontfix for 1.0.
But I've added a fix for it in 2.0 (that I hope is coming out soon).

<hr>

Until 2.0 is out, I think removing the word "Address" and putting the address on the left might be a good solution.

![image](https://user-images.githubusercontent.com/7156344/45911396-00b02500-be13-11e8-9deb-ca4825688351.png)

**To make this change:**
- Find the line with address (`include/header.php` line `142`):
https://github.com/Wruczek/ts-website/blob/d5a99b430a76d4abe5662de142d2603c091a6ca6/include/header.php#L142
- Replace it with: 
```html
<p><i class="fa fa-globe fa-fw" aria-hidden="true"></i> <a href="ts3server://<?php echo $config['teamspeak']['displayip']; ?>" style="float: none"><?php echo $config['teamspeak']['displayip']; ?></a></p>
```
<br>
Let me know if that answers it :)

**Comment by ShadyShell** [comment]
That worked perfectly, thank you very much :). Do you want me to leave this open for tracking on 2.0 or close it?

**Comment by Wruczek** [comment]
Its already fixed in 2.0 so we can close it.
